client/lib/create-selector/test/index.js

```diff
@@ -1,4 +1,5 @@
 /** @format */
+/* eslint-disable no-console */
```

Comment:
As in the per line disables?
It crossed my mind but went with a global rule since `console` is used so much here.
I'd guess there'll be more tests coming that focus on  `console.warn` soon too (Marek is planning to warn against missing out `getDependants`) so we could be adding a fair amount of noise.
Could be worth the trade off though, for strictness :)

Reply:
makes sense -- your call :)